modules/references/tests/server/reference-create.server.routes.tests.js

```diff
@@ -72,6 +74,7 @@ describe('Create a reference', () => {
                 hostedThem: true,
               },
               recommend: 'yes',
```

Comment:
should this still be included?

Reply:
Yes, as the user will still submit that info, it's just that we won't return it.